src/client.ts

```diff
@@ -369,6 +446,36 @@ export class Client {
   // FILE MARKER - METHODS - PRIVATE ///////////////////////////////////////////
   //////////////////////////////////////////////////////////////////////////////
 
+  /**
+   * This method is used internally to calculate the element Viewport (Dimensions)
+   * @param selector
+   * @returns ViewPort object
+   */
+  private async getViewport(selector: string) {
```

Comment:
Updated docblocks and method return type please :)

Reply:
I added some docs, please do let me know if i can add something more